packages/core/src/parsers.ts

```diff
@@ -1,21 +1,13 @@
-import { CCSSParser, CCSSPipe } from '@/types'
+import { CCSSInput, CCSSParser, CCSSPipe } from '@/types'
 
-export const toCSSRule = (cssProp, objectProp) => {
-    return (input, prop, options) => {
-        return options.outputTransformer.toCSSRule(cssProp, objectProp, input, prop, options)
-    }
-}
+export const toCSSRule = (cssProp: string, objectProp: string): CCSSParser => (input, prop, options) =>
+    options.outputTransformer.toCSSRule(cssProp, objectProp, input, prop, options)
 
-export const parseSingle: CCSSParser = (input, prop, options) =>
+export const parseSingle: CCSSParser = (input, _, options) =>
     typeof input === 'number' ? (input === 0 ? 0 : options.applyUnit(input)) : input
 
-const applyArray: CCSSParser = (input, prop, options) => {
-    let out = ''
-    for (const i of input) {
-        out += `${parseSingle(i, prop, options)} `
-    }
-    return out
-}
+const applyArray: CCSSParser = (input, prop, options) =>
+    input.reduce((acc: string, curr: CCSSInput) => acc.concat(`${parseSingle(curr, prop, options)} `), '')
```

Comment:
Stop rewriting code to slower versions. These are highly optimized codes everywhere. I spent several hours with JS benchmarks.

Reply:
I remove this and I'm using original version instead.